diff --git a/dist/index-Bihpwn75.d.cts b/dist/index-Bihpwn75.d.cts
index 04d4ffcf245e38e0f4cba27e606d856a6c1dd667..b2be0f971628085fe9766fccdb8eaa4800144e29 100644
--- a/dist/index-Bihpwn75.d.cts
+++ b/dist/index-Bihpwn75.d.cts
@@ -58,6 +58,16 @@ interface CreateThemedStyled extends CreateStyled {
 declare const styled: CreateThemedStyled;
 declare namespace index_d_exports {
   export { CSSObject$1 as CSSObject, CacheProvider, EmotionThemeContext, EmotionThemeProvider, Global, PaletteColor, Theme$1 as Theme, ThemeCustomizationOptions, createShouldForwardProp, css, keyframes, props, styled, useTheme };
+  export import BackgroundProps = import__soroush_tech_styled_system.BackgroundProps;
+  export import BorderProps = import__soroush_tech_styled_system.BorderProps;
+  export import FlexboxProps = import__soroush_tech_styled_system.FlexboxProps;
+  export import GridProps = import__soroush_tech_styled_system.GridProps;
+  export import LayoutProps = import__soroush_tech_styled_system.LayoutProps;
+  export import PaddingProps = import__soroush_tech_styled_system.PaddingProps;
+  export import PositionProps = import__soroush_tech_styled_system.PositionProps;
+  export import ResponsiveValue = import__soroush_tech_styled_system.ResponsiveValue;
+  export import SpaceProps = import__soroush_tech_styled_system.SpaceProps;
+  export import TypographyProps = import__soroush_tech_styled_system.TypographyProps;
 }
 import * as import__soroush_tech_styled_system from "@soroush.tech/styled-system";
 //#endregion
diff --git a/dist/index-Dm9HfgV6.d.mts b/dist/index-Dm9HfgV6.d.mts
index b6ca48cd450a9a43cde6a283012bb38477ae4bc8..289acffe1f3a576c7b0685014d6e0a8d4cc8c882 100644
--- a/dist/index-Dm9HfgV6.d.mts
+++ b/dist/index-Dm9HfgV6.d.mts
@@ -57,6 +57,16 @@ interface CreateThemedStyled extends CreateStyled {
 declare const styled: CreateThemedStyled;
 declare namespace index_d_exports {
   export { CSSObject$1 as CSSObject, CacheProvider, EmotionThemeContext, EmotionThemeProvider, Global, PaletteColor, Theme$1 as Theme, ThemeCustomizationOptions, createShouldForwardProp, css, keyframes, props, styled, useTheme };
+  export import BackgroundProps = import__soroush_tech_styled_system.BackgroundProps;
+  export import BorderProps = import__soroush_tech_styled_system.BorderProps;
+  export import FlexboxProps = import__soroush_tech_styled_system.FlexboxProps;
+  export import GridProps = import__soroush_tech_styled_system.GridProps;
+  export import LayoutProps = import__soroush_tech_styled_system.LayoutProps;
+  export import PaddingProps = import__soroush_tech_styled_system.PaddingProps;
+  export import PositionProps = import__soroush_tech_styled_system.PositionProps;
+  export import ResponsiveValue = import__soroush_tech_styled_system.ResponsiveValue;
+  export import SpaceProps = import__soroush_tech_styled_system.SpaceProps;
+  export import TypographyProps = import__soroush_tech_styled_system.TypographyProps;
 }
 import * as import__soroush_tech_styled_system from "@soroush.tech/styled-system";
 //#endregion
